app/models/user.rb

```diff
@@ -6,4 +6,5 @@ class User < ApplicationRecord
 
   has_many :posts
   has_many :comments
+  has_many :votes
```

Comment:
what do you think about a `dependent: :destroy` being set on all of these associations?